apps/vaporgui/SettingsParams.cpp

```diff
@@ -84,7 +84,7 @@ SettingsParams::SettingsParams(
 ) : ParamsBase(ssave, _classType) {
 
 	_settingsPath = QDir::homePath().toStdString();
-	_settingsPath += QDir::separator().toAscii();
+    _settingsPath += "/";
```

Comment:
Does this separator work in Windows?

Reply:
It does, but I agree that this can be improved.  Done.